wizeline/wize-q-remix
# feat: add cypress configuration and initial tests

#### What does this PR do?
- Adds Cypress project into the project for E2E testing. This is run with Github Actions
- Adds some base tests for home page. There are some modifications needed in components to be able to create more tests. Mainly adding unique identifiers so that Cypress can find these.
- Modifies integration test database setup and teardown commands into separate files, so that Cypress can use this on demand
- Modifies root jsconfig file into a tsconfig file, which was needed for Cypress support. This means we can now have typescript files in the project in the future, but on the meantime we can continue using Javascript
- Adds libraries, and eslint changes for cypress

#### Where should the reviewer start?
- Install latest packages with: `npm i`

#### How should this be manually tested?
- See Github Action tests execution
- Pull changes locally and run: `npm run test:cypress:open`


#### What are the relevant tickets?
Closes #7
